### ISSUE TITLE
Dosen't work with macOS 10.12

### ISSUE THREAD

**Issue by leoSince2017** [noma4i/NativeNTFS-OSX#1]
I'm so glad to use your shell script to mount my ntfs to R/W. But unfortunately, maybe it's not adapted to the newest os x. I'm sure I've used 'sudo' to run it.

that's what echoed in the terminal:
Volume Data1 on disk2s6 unmounted
Volume Data1 on /dev/disk2s6 mounted

My disk(on the desktop) disapeared at once and then showed up. But I just can't write it.
Whatever, thank you!

**Comment by noma4i** [comment]
Hey! Thanx for stoping by. This script is in general wrapper for diskutil. Could you perform this manually:
`diskutil list` - to find your volume
`diskutil mount /path/to/volume` - where path to volume is taken from
This should mount volume in read-only mode. If that happens use script to mount as read-write.

**Comment by leoSince2017** [comment]
I've got what's wrong !
I read some essays about that and opened /etc/fstab using vim /etc/fstab
and I found that the line in this file don't looks like the format what told in that essays.
There's something more in the fstab. Between 'UUID=xxxxxxx' and 'none', there is another 'xxxxxxx' as long as 'UUID'.
I tried to delete what's more than it told, and it worked.

I don't know how to write shell script, so I don't know what happened.
But it might because of apple have updated something into the system in the 3 years since you stop updating this script.😂 Or maybe my hard drive is not as same as others😂.

Thank you again!
